### PR TITLE
Gradle installation note <2.0.x> [9882]

### PR DESCRIPTION
**Pull request by imontesino**
Signed-off-by: Ignacio Montesino <ignaciomontesino@eprosima.com>

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/655/)

File: docs/installation/sources/sources_linux.rst [hunk 1]
```diff
@@ -318,6 +318,15 @@ Once the requirements above are met, compile *Fast DDS-Gen* by following the ste
     cd Fast-DDS-Gen
     gradle assemble
 
+.. note::
+
+    If errors occur during compilation or you do not wish to install gradle, an executable script is included which will
+    download a gradle temporarily for the compilation step.
```
Comment (EduPonz) [inline docs/installation/sources/sources_linux.rst:324]: Same for MAC
```suggestion
    If errors occur during compilation, or *Gradle* installation is wished to be avoided, an executable script is included which will
    download a temporal *Gradle* for the compilation step.
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

**Comment by EduPonz** [comment]
This PR requieres rebasing

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/672/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/673/)

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM

**Review by EduPonz — APPROVED** [review]
LGTM

**Pull request merged**